src/image/image.js

```diff
@@ -69,7 +69,7 @@ import omggif from 'omggif';
  * let img = createImage(66, 66);
  * img.loadPixels();
  * let d = pixelDensity();
- * let halfImage = 4 * (img.width * d) * (img.height / 2 * d);
+ * let halfImage = 4 * (img.width * d) * ((img.height / 2) * d);
```

Comment:
This is not passing linting and should be reverted

Reply:
that's interesting because on the other hand, the linter would not let me commit unless I made that change... could it be that i have another version of the linter installed? I think I am using the exact setup that the contributor_docs mentions, but clearly i missed something here

Reply:
Hmm...I double check that locally.

Reply:
Made sure to setup and run everything again just in case I missed something but here is the output from the log:

```
[18:21:29] Running tasks for src/**/*.js [failed]
[18:21:29] → ✖ node --require @babel/register ./utils/sample-linter.js found some errors. Please fix them and try committing again.
/Users/jesi/repositorios/p5.js/src/image/image.js:72:43: Replace `img.height·/·2` with `(img.height·/·2)` [Error/prettier/prettier]
```

You can see it is asking for the change I made. That said, this only happens on commit, but this is not triggering when `npm run dev` is running and the console shows other errors. 

Are the linters in each system different?

Reply:
I've tested locally and the original version works while the one in the PR failed as the CI test did. Does running `npm test` clear for you? Do you have prettier separately installed on your system somewhere that is of a different version than what is used here?

Reply:
sorry for the delay! did not get notified.

indeed, if I run npm test with my change, it prompts an error removing exactly that.

I changed it back to what it should be and now everything in `npm test` runs fine.